docs/topics/mta-rn-new-features-6-2-0.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
+
+.Integration with JIRA
+A new _Jira_ entry has been added in the left sidebar, under the Control Menu. This feature enables the user (Administrator, Architect) to create new Jira instances and edit existing instances.
+
+.Report custom metrics on MTA usage back through OpenShift Telemetry
+MTA reports back through OpenShift Telemetry as to the applications that it processes:
+
+* Number of applications in Inventory
+* Number of Analysis Run (aggregated)
+* Number of Assessments Run
+
```

Comment:
@HagayVider1 
The details of this feature do not belong in the MTA documentation. We should only mention the following as a new feature:
".OpenShift Monitoring integration
This release supports OpenShift Monitoring integration."